### ISSUE TITLE
Master Data Type Library export doesn't include versions

### ISSUE THREAD

**Issue by csnewman88** [haffonist/igamt#2680]
When exporting a master data type library to Word, the output doesn't include the versions the data type flavor applies to. Particularly in the master data type library (but probably also in user libraries) there will be multiple versions represented and it's hard to know which flavors are appropriate for different base versions.

The export should include the relevant versions for a flavor.

![image](https://user-images.githubusercontent.com/17408249/33383772-c6c1a674-d4e9-11e7-96a2-26b106586865.png)

**Comment by afrold** [comment]
@maxence-lefort for you. After upgrade.

**Comment by maxence-lefort** [comment]
@haffonist should we add the version for both the datatype library and ig doc exports?

**Comment by afrold** [comment]
yes. Data type library export format should be the same as the iG one.
